### ISSUE TITLE
Add type field

### ISSUE THREAD

**Issue by ligi** [ethereum-lists/chains#3]
wonder if we should add a type field - POW,POA,POS

**Comment by pedrouid** [comment]
👍

**Comment by bmann** [comment]
Should this be ```consensus_type``` -- eg. there is Cosmos Tendermint etc

**Comment by pedrouid** [comment]
Agreed, `consensus_type` makes more sense. However I would be skeptical to add Cosmos Tendermint for example because it's not EVM-based.

**Comment by bmann** [comment]
If the line in the sand is “runs a compatible EVM” — I think that’s a useful filter. 

We could post what we think the filter is to explain this list.

**Comment by irajsava** [comment]
_[]()_
